template.md

```diff
@@ -0,0 +1,305 @@
+---
+# required metadata
+
+title: [ARTICLE TITLE | SERVICE NAME]
+description:
+keywords:
+author: [GITHUB USERNAME]
+manager: [MANAGER ALIAS]
```

Comment:
@svick good point. None of us are particularly happy that this needs to be included at all. We're including one person as the manager to keep things simple. I think the default (for community folks) should be "dotnet-bot".